.travis.yml

```diff
@@ -1,29 +1,19 @@
+dist: bionic
 language: python
-dist: bionic   # required for Python >= 3.7
 cache: pip
-
-# Environment changes have to be manually synced with 'tox.ini'.
-# See: https://github.com/travis-ci/travis-ci/issues/3024
-
-python:
-  - "3.6"
-  - "3.7"
-
 matrix:
   include:
-    - python: 3.5
+    - python: "3.5"
       dist: xenial  # Bionic has no Python 3.5
-
+    - python: "3.6"
+    - python: "3.7"
     # Disabled, see https://github.com/sybrenstuvel/python-rsa/issues/131
     #- python: pypy3.5
     #  dist: xenial  # Bionic has no Python 3.5
-
+before_install: pip install --upgrade pip
 install:
   - pip install poetry
   - poetry install -v
-
-script:
-  - poetry run py.test tests/
-
-after_success:
-  - poetry run coveralls
+before_script: flake8 .
```

Comment:
Is flake8 installed by default? Personally I'd rather run it with `poetry run flake8 .`, so that it's running the same version in Travis-CI as you'd get when running locally. Makes things a bit easier to control.

Reply:
poetry is installing flake8.

If poetry is also going to run flake8 then this line can be removed.  I am not a user of poetry (yet).

Reply:
But without `poetry run` in front of it, it's not going to magically run from the Poetry-managed virtualenv.